Makefile

```
@@ -0,0 +1,6 @@
+TEST_DIR := ./agent/functions
+
+.PHONY: test
+
+test:
+	cd $(TEST_DIR) && go test -v
```

from SecurityExpert
Makefileにおいて、特にセキュリティ上の問題は見当たりませんが、`cd`コマンドを使用する際には、ディレクトリの存在を確認するためのチェックを追加することをお勧めします。これにより、意図しないディレクトリへの移動を防ぐことができます。

```suggestion
test:
	[ -d $(TEST_DIR) ] && cd $(TEST_DIR) && go test -v
```